### PR TITLE
Update v1.8.0

### PR DESCRIPTION
**Pull request by agourlay**

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "qdrant-client"
-version = "1.6.0"
+version = "1.7.0"
```
Comment (timvisee) [inline Cargo.toml:3]: Not 1.8?

Comment (agourlay) [reply Cargo.toml:3]: I had the impression the actual version update is done when pushing to master.

But I can change it if it is not the case :)

Comment (agourlay) [reply Cargo.toml]: decided to fix it right away!

**Comment by agourlay** [comment]
@timvisee docker image 1.8.0 pulled and tests green :+1:

File: Cargo.toml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "qdrant-client"
-version = "1.6.0"
```
[the remark this answers is not in the document]
Comment (agourlay) [reply Cargo.toml]: done https://github.com/qdrant/rust-client/pull/88/commits/e046a3c4ba4b05d90d5fd21c13a3d4c9e056fafb

**Review by timvisee — APPROVED** [review]

**Pull request merged**